cli/utils.py

```diff
@@ -29,3 +30,6 @@ def extractShaFromImage(self, image):
             if len(sha) != 0:
                 return sha[-1]
         return ''
+
+    def getStatsClient(self):
+        return statsd.StatsClient('statsd.roger.dal.moz.com', 8125)
```

Comment:
Moved the statsd endpoint and port to "roger-env" in roger-cli. Now reading these values from roger-env.